packages/smart-contracts/src/contracts/Erc20ConversionProxy.sol

```diff
@@ -19,14 +19,24 @@ contract Erc20ConversionProxy {
     chainlinkConversionPath = ChainlinkConversionPath(_chainlinkConversionPathAddress);
   }
 
-  // Event to declare a transfer with a reference
+  // Event to declare a conversion with a reference
   event TransferWithConversionAndReference(
     uint256 amount,
     address currency,
     bytes indexed paymentReference,
     uint256 feeAmount,
     uint256 maxRateTimespan
   );
+  
+  // Event to declare a transfer with a reference
+  event TransferWithReferenceAndFee(
```

Comment:
This event is sent by the contract as a delegate of ERC20FeeProxy. Declaring it here will help reading events out of future blockchains after code publication.